README.md

```diff
@@ -1,3 +1,79 @@
-# Ansible Collection - a10.acos_collection
+# Download the Collection Using following Command 
 
-Documentation for the collection.
\ No newline at end of file
+` ansible-galaxy collection install a10.acos_collection ` 
+
+-----------------------------------------------------------------------------------------------------
+
+## How to Use Collection : 
+
+A] Set the plugin path using any one of the following way:
+	
+	1) Copy/Move terminal and cli_conf plugin inside 
+	   ~/.ansible/plugins 			
+				OR 
+	   /usr/share/ansible/plugins folder
+	
+	2) Add below two lines in /etc/ansible/ansible.cfg File 
+
+		cliconf_plugins  = <collection-dir-path>/a10/acos_collection/plugins/cliconf
+		terminal_plugins = <collection-dir-path>/a10/acos_collection/plugins/terminal
+	
+	3) Export following environment variables for new session
+
+		export ANSIBLE_CLICONF_PLUGINS=<collection-dir-path>/a10/acos_collection/plugins/cliconf
+		export ANSIBLE_TERMINAL_PLUGINS=<collection-dir-path>/a10/acos_collection/plugins/terminal
+ 	
+ 	4) Save this variable in .bashrc File 
+
+		export ANSIBLE_CLICONF_PLUGINS=<collection-dir-path>/a10/acos_collection/plugins/cliconf
+		export ANSIBLE_TERMINAL_PLUGINS=<collection-dir-path>/a10/acos_collection/plugins/terminal
+
+-----------------------------------------------------------------------------------------------------
+
+B] Accessing/Using Collection (Use Any one of the way ) :
+	
+	1) Copy/Move - Collection folder we got from tarball inside 
+					~/.ansible/collections 
+							OR 
+					/usr/share/ansible/collections folder 
+	
+	2)  Export following environment variables for new session
+		ANSIBLE_COLLECTIONS_PATHS=<path-to-collections-folders>
+	
+	3) Add below line in /etc/ansible/ansible.cfg File 
+		
+		collections_paths=<path-to-collection1>:<path-to-collection2>
+
+	4) Keep your playbooks to run in relative to collection 
+
+			|── myplaybook.yml
+			├── collections/
```

Comment:
This is just an example that we have used from Ansible collection documentation. It has so many directories because it structures of ansible collection which we get after installing it